src/components/RadioButton/RadioButton.scss

```diff
@@ -1,6 +1,7 @@
 @import "../../styles/themes";
```

Comment:
please check this file and don use static variables,  use from mixin or global.css